jetty-webapp/src/main/java/org/eclipse/jetty/webapp/ClasspathPattern.java

```diff
@@ -719,9 +714,8 @@ public boolean match(String name, URL url)
             {
                 return URIUtil.getJarSource(url.toURI());
             }
-            catch (URISyntaxException e)
+            catch (URISyntaxException ignored)
             {
-                LOG.ignore(e);
```

Comment:
I wonder if we could get by with something like:
```java
if (LOG.isDebugEnabled())
   ignored.printStackTrace();
```

Reply:
in order to fix the issue  we don't want to load the Logger class